.travis.yml

```diff
@@ -46,8 +46,8 @@ after_success:
 - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then chmod +x $TRAVIS_BUILD_DIR/createtarball.sh && $TRAVIS_BUILD_DIR/createtarball.sh $DEPLOY_DIR $OUT_PKG $SRC_DIR; fi
 
 deploy:
-provider: bintray
-file: $TRAVIS_BUILD_DIR/deploybintray.json
-user: dpfoose
-key:
-    secure: CkzrPKCuAAQxnPSEXbtIbJzjFP1dDDQs0KdyxA10KvTapEOp/JppsUd2BQ7PRSIU1vdvLZM7ysnpMWsjAZQ2+MR5HxBBKRjgqjwy93XFgXS+RP36AagQIEhlu+aqvbDl9/cgMAxrj57lizQA2KUF+zIkonKByDmVNc6oExlUq0WfeCNQZ+RBb7aPfckVdViMsTtfP/3S56cY4kgcc+uVx9Au/YXTB+BHFiON7TL40CLm+FW3HArvBrmIQIBDNE50JfSBe+jCk2/wPFZnsv0FH6KcnT32cNiT2b8gUltOpHWoPbh2y5cRur6NgaAg+WaXy3XIb2KhphkA4ih5WhXGgJ+6X7V+DX1lJd/2abSJzstI+RfPnuIOxP4KjijjdhT96Gq9OLVAYT7VUcM+IyWmu/0PJCWlnFb03uJRos4QJ05QV8FxWD8K80TwfN3BIPvPwlN+uIaTsuGV6LSXIpj8zrBToM8r990kbDunxEAPBKZEP9SCpUT/26UPOp0ON2oMRSAgcFOw7c0nz+fkShWqzlaQ/VXZL4/4TYlga3ZGGWPDiEMQ0GUKZHylZZ11SrI4kuHbspGCBTb5xMrFqJ2pI+zUlPKSsInnDUHa+VDAUb0FoTwX/wSBnO6pQvT19LQLGIORMfOhTYARmz+6ChEGr4X5iO6+PQDGe95oJzy4tVI=
+    provider: bintray
+    file: $TRAVIS_BUILD_DIR/deploybintray.json
+    user: dpfoose
+    key:
+        secure: CkzrPKCuAAQxnPSEXbtIbJzjFP1dDDQs0KdyxA10KvTapEOp/JppsUd2BQ7PRSIU1vdvLZM7ysnpMWsjAZQ2+MR5HxBBKRjgqjwy93XFgXS+RP36AagQIEhlu+aqvbDl9/cgMAxrj57lizQA2KUF+zIkonKByDmVNc6oExlUq0WfeCNQZ+RBb7aPfckVdViMsTtfP/3S56cY4kgcc+uVx9Au/YXTB+BHFiON7TL40CLm+FW3HArvBrmIQIBDNE50JfSBe+jCk2/wPFZnsv0FH6KcnT32cNiT2b8gUltOpHWoPbh2y5cRur6NgaAg+WaXy3XIb2KhphkA4ih5WhXGgJ+6X7V+DX1lJd/2abSJzstI+RfPnuIOxP4KjijjdhT96Gq9OLVAYT7VUcM+IyWmu/0PJCWlnFb03uJRos4QJ05QV8FxWD8K80TwfN3BIPvPwlN+uIaTsuGV6LSXIpj8zrBToM8r990kbDunxEAPBKZEP9SCpUT/26UPOp0ON2oMRSAgcFOw7c0nz+fkShWqzlaQ/VXZL4/4TYlga3ZGGWPDiEMQ0GUKZHylZZ11SrI4kuHbspGCBTb5xMrFqJ2pI+zUlPKSsInnDUHa+VDAUb0FoTwX/wSBnO6pQvT19LQLGIORMfOhTYARmz+6ChEGr4X5iO6+PQDGe95oJzy4tVI=
```

Comment:
indentation is important in YAML files. I believe this is why they didn't get bintray didn't authorize the deployment before.